hcloud/storage_box.go

```diff
@@ -277,6 +297,8 @@ func (o StorageBoxFoldersOpts) values() url.Values {
 }
 
 // Folders lists folders in a [StorageBox].
+//
+// See https://docs.hetzner.cloud/reference/hetzner#storage-boxes-list-content-of-storage-box
```

Comment:
I think the slug changed in our internal documentation (no published yet), consider using the updated link.